### ISSUE TITLE
Save for later is unable to open files that are stored locally

### ISSUE THREAD

**Issue by miam-miam** [allen505/save-for-later-firefox#9]
Save for later is unable to open files that are stored locally and provides no response showing that the extension cannot do it. Is this just due to the fact that the api does not allow you to open local files?

**Comment by allen505** [comment]
Hi @miam-miam100 , could you please share the URL to the file you are trying to save. The type of URLs saved by Save for Later are given in [this line](https://github.com/allen505/save-for-later/blob/master/popup/warehouse.js#L6). It is a regex pattern used to match appropriate URLs.

**Comment by miam-miam** [comment]
Sure it seems to me to only be a normal file but I may be wrong:
file:///D:/Documents/EPQ/Quantum_Computing_for_Computer_Scientists_-_Yanofsky-cup-2008.pdf

**Comment by allen505** [comment]
Yes, I see your point @miam-miam100 . The extension saves the links but it does not open it. To open a local file the API used is different. This is definitely a bug on the extension's side.  
Possible solution to open local files is in [this Stackoverflow answer](https://stackoverflow.com/a/44516256) . Implementing this in the current code though seems a bit tricky. We'd have to distinguish local vs web links before opening it in [this line](https://github.com/allen505/save-for-later/blob/master/popup/warehouse.js#L49)
